Accessibilty_Scanner_IAM (#9899)
braze-inc/braze-docs: _docs/_help/accessibility.md

<div aria-live="polite">Your preferences have been saved.</div>
```
{% endraw %}

## Automated accessibility testing

To help you identify and fix accessibility issues early, Braze offers automated accessibility testing in the following areas:

- [Inbox Vision]({{site.baseurl}}/user_guide/message_building_by_channel/email/inbox_vision/#accessibility-testing) for emails

> - [Accessibility scanner]({{site.baseurl}}/user_guide/message_building_by_channel/in-app_messages/testing/#accessibility-scanner) for messages created using our HTML editor (this would appear in HTML In-App Messages, HTML Content Blocks, Email footer composer, Email preference center composer).

Reviewer: ```suggestion
- [Accessibility scanner]({{site.baseurl}}/user_guide/message_building_by_channel/in-app_messages/testing/#accessibility-scanner) for messages created using our HTML editor (For example, when creating HTML in-app messages, HTML Content Blocks, [custom email footers]({{site.baseurl}}/user_guide/message_building_by_channel/email/custom_email_footer), or [email preference centers]({{site.baseurl}}/user_guide/message_building_by_channel/email/preference_center/dnd_preference_center).
```

> - [Accessibility scanner]({{site.baseurl}}/user_guide/message_building_by_channel/in-app_messages/testing/#accessibility-scanner) for messages created using our HTML editor (For example, when creating HTML in-app messages, HTML Content Blocks, [custom email footers]({{site.baseurl}}/user_guide/message_building_by_channel/email/custom_email_footer), or [email preference centers]({{site.baseurl}}/user_guide/message_building_by_channel/email/preference_center/dnd_preference_center).

Reviewer: ```suggestion
- [Accessibility scanner]({{site.baseurl}}/user_guide/message_building_by_channel/in-app_messages/testing/#accessibility-scanner) for messages created using our HTML editor (for example, HTML in-app messages, HTML Content Blocks, [custom email footers]({{site.baseurl}}/user_guide/message_building_by_channel/email/custom_email_footer), [email opt-in pages]({{site.baseurl}}/user_guide/message_building_by_channel/email/opt_in_pages), and [email unsubscribe pages]({{site.baseurl}}/user_guide/message_building_by_channel/email/unsubscribe_pages)).
```

These tests check your message against the Web Content Accessibility Guidelines ([WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/)) standard—a set of internationally recognized technical standards for accessible content. Any issues that can be detected automatically are flagged and categorized by severity to help you prioritize.

{% alert note %}

> Inbox Vision works for both HTML and drag-and-drop emails. The in-app message scanner only runs on messages built with the custom HTML editor.

Reviewer: Thanks both! Quick clarification, Isn't [preference center drag-and-drop](https://www.braze.com/docs/user_guide/message_building_by_channel/email/preference_center/dnd_preference_center)?
